config/settings.yml

```diff
@@ -1378,6 +1378,7 @@ vanotify:
         notice_of_disagreement_form_error_email: fake_nod_template_id
         supplemental_claim_evidence_error_email: fake_sc_evidence_template_id
         supplemental_claim_form_error_email: fake_sc_template_id
+        supplemental_claim_secondary_form_error_email: fake_sc_secondary_form_template_id
```

Comment:
Will you be adding the real secondary form template id to the manifest repo like the other template IDs? https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/tree/main/apps/vets-api

Reply:
Yes, working on that PR this morning, we're having trouble getting through login at VA Notify to grab the template IDs at the moment

Reply:
First settings PR: 84372-decision-review-4142-email-template-settings

Reply:
Template id PR was merged yesterday.